From 69201f8da7356fb62857a76e19dfbfbdb69e8e36 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 8 Jun 2015 13:54:59 +0200
Subject: [PATCH 161/310] oidentd: fix gcc5 build

gcc5 uses -std=gnu11 instead of -std=gnu89
by default

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/oidentd/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/oidentd/Makefile b/net/oidentd/Makefile
index 8cebb1d..e29c3b3 100644
--- a/net/oidentd/Makefile
+++ b/net/oidentd/Makefile
@@ -28,6 +28,8 @@ define Package/oidentd/description
 	A nat aware identd server that will correctly forwared requests to clients.
 endef
 
+TARGET_CFLAGS+= -std=gnu89
+
 # uses GNU configure
 
 define Build/Compile
-- 
2.5.0

